tests/snippets/stdlib_os.py

```diff
@@ -73,3 +75,26 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 	assert os.read(fd, len(CONTENT2)) == CONTENT2
 	assert os.read(fd, len(CONTENT3)) == CONTENT3
 	os.close(fd)
+
+	fname2 = tmpdir + os.sep + FILE_NAME2
```

Comment:
We are in need of `os.path.join` :).

Reply:
It's all python code in path. I will look if it can be copied as is.